data/events/scripts/player.lua

```diff
@@ -93,13 +93,43 @@ function Player:onMoveCreature(creature, fromPosition, toPosition)
 	return true
 end
 
-function Player:onReport(message, position, category)
+function Player:onReportRuleViolation(targetName, reportType, reportReason, comment, translation)
+	local name = self:getName()
+	local pendingReport = function () local f = io.open(string.format("data/reports/players/%s-%s-%d.txt", name, targetName, reportType), "r") ; if f then io.close(f) return true else return false end end
```

Comment:
@ranisalt isn't better just indent this function instead? since there are only player event functoins on this file